.github/workflows/sync-design-tokens.yml

```diff
@@ -0,0 +1,33 @@
+name: Design tokens
+
+on:
+  schedule:
+    - cron: "0 8 * * *"
+
+jobs:
+  synchronise:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.5
+        env:
+          LANG: en_US.UTF-8
+      - name: Generate json files
+        env:
+          URL: https://raw.githubusercontent.com/scaleway/design-tokens/main/tokens.json
+        run: |
+          curl ${URL} | jq 'reduce (.dark.shades | to_entries | .[]) as $sentiment ({}; . + {"\($sentiment.key)": (reduce($sentiment.value | to_entries | .[]) as $shade ({}; . + { "\($shade.key)": $shade.value.value })) }) | {"colors": .}' > src/theme/tokens/dark.json
+          curl ${URL} | jq 'reduce (.light.shades | to_entries | .[]) as $sentiment ({}; . + {"\($sentiment.key)": (reduce($sentiment.value | to_entries | .[]) as $shade ({}; . + { "\($shade.key)": $shade.value.value })) }) | {"colors": .}' > src/theme/tokens/light.json
+
+      - name: Compare & Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          base: main
+          author: Scaleway Design Bot <design@scaleway.com>
```

Comment:
Needs to be github@scaleway.com to be associated with https://github.com/scaleway-bot :)